contracts/src/periphery/EigenDACertVerifierV1.sol

```diff
@@ -0,0 +1,152 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {IEigenDACertVerifierV1} from "src/periphery/interfaces/IEigenDACertVerifierV1.sol";
+import {IEigenDAThresholdRegistry} from "src/interfaces/IEigenDAThresholdRegistry.sol";
+import {IEigenDABatchMetadataStorage} from "src/interfaces/IEigenDABatchMetadataStorage.sol";
+import {EigenDACertVerificationV1Lib as CertLib} from "src/periphery/libraries/EigenDACertVerificationV1Lib.sol";
+import "src/interfaces/IEigenDAStructs.sol";
+
+/**
+ * @title CertVerifierV1 - EigenDA V1 certificate verification
+ * @author Layr Labs, Inc.
+ * @notice Contains all V1-specific verification functionality
+ */
+contract EigenDACertVerifierV1 is IEigenDACertVerifierV1 {
+    /// @notice Thrown when there is a length mismatch
```

Comment:
@samlaf #1472 

length mismatch between what and what? cert fields and some stuff stored onchain? or..

Reply:
This error is currently only used in verifyDaCertsV1, since we need to verify that the length of blob headers and proofs is the same. I'll add this to the comment.

Reply:
https://github.com/Layr-Labs/eigenda/pull/1477/commits/c7963d237e43d0cb722f1bad8bbe4fef35d933a4

Reply:
I'm confused by the change. You deleted the function?